src/content/releases/v0.4.0.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: "Docs & Files, New Homepage, Redesigned Space Dashboard, and more"
+version: "0.4"
+date: 2025-02-03
+description: "Operately v0.4 introduces Docs & Files, simplifies the homepage and space dashboard, unifies Goals & Projects, and introduces new roles for company administration."
+published: true
+---
+
+import BigImage from '../../components/BigImage.astro';
+import docsFiles from './assets/v040/docs-files.png';
+import homepage from './assets/v040/homepage.png';
+import spaceDashboard from './assets/v040/space-dashboard.png';
+import unifiedGoalsAndProjects from './assets/v040/unified-goals-and-projects.png';
+import adminsOwners from './assets/v040/admins-owners.png';
+import drafts from './assets/v040/drafts.png';
+
+Operately v0.4 is here, packed with improvements that make your Operately
+experience even better. In this release, we're focusing on document management, 
+improving the homepage and space dashboards, and introducing new roles for 
+account administration.
+
+## Docs & Files: A central hub for all your work documents
+
+Every space now has a dedicated Docs & Files section. Upload, organizate, and
```

Comment:
Since "Upload" is the first word, it kind of leads to a conclusion that this is a file upload only feature? Consider "Create, upload, and organize..."

not "organizate" ;)